features/registering_pages.feature

```diff
@@ -146,3 +146,26 @@ Feature: Registering Pages
     And I follow "Check"
     Then I should see the content "Chocolate lØves You Too!"
     And I should see the Active Admin layout
+
+  Scenario: Registering a page with paginated index table for a collection Array
+    Given a user named "John Doe" exists
+    Given a configuration of:
+    """
+    ActiveAdmin.register_page "Special users" do
+      content do
+        collection = Kaminari.paginate_array(User.all).page(1)
```

Comment:
I just wanted to produce an Array. But I agree, might be better if it represents a real use case. Chnaged it.